src/host/snmalloc.cpp

```diff
@@ -7,5 +7,4 @@
 #  define NDEBUG
 #endif
 
-#include "snmalloc/src/override/malloc.cc"
-#include "snmalloc/src/override/new.cc"
```

Comment:
I think it would be better to keep 
```
#include "snmalloc/src/override/new.cc"
```
and drop
```
#include "snmalloc/src/override/malloc.cc"
```
The `new.cc` file includes `malloc.cc`, but also provides overrides for the `new` and `delete` operations, so they can get memory with one fewer function call.